Interface/SamuraiHudWindow.cs

```diff
@@ -0,0 +1,155 @@
+        
+        public SamuraiHudWindow(DalamudPluginInterface pluginInterface, PluginConfiguration pluginConfiguration) : base(pluginInterface, pluginConfiguration) { }
+
+        protected override void Draw(bool _) {
+            DrawHealthBar();
+            DrawPrimaryResourceBar();
+            DrawSenResourceBar();
+            DrawMeditationResourceBar();
+            DrawTargetBar();
+        }
+
+        protected override void DrawPrimaryResourceBar()
+        {
+            var gauge = PluginInterface.ClientState.JobGauges.Get<SAMGauge>();
+
+            // const int xPadding = 5;
+            const int yPadding = 3;
+            var barWidth = BarWidth;
+            var xPos = CenterX - XOffset;
+            var yPos = CenterY + YOffset - BarHeight - yPadding;
+            var cursorPos = new Vector2(xPos, yPos);
+            const int chunkSize = 50;
+            var barSize = new Vector2(barWidth, BarHeight);
+
+            // Chunk 1
+            var Kenki = Math.Min((int)gauge.Kenki, chunkSize);
+            var scale = (float)Kenki / chunkSize;
+            var drawList = ImGui.GetWindowDrawList();
+            drawList.AddRectFilled(cursorPos, cursorPos + barSize, 0x88000000);
+
+            if (scale >= 1.0f)
+            {
+                drawList.AddRectFilledMultiColor(
+                    cursorPos, cursorPos + new Vector2(barWidth * scale, BarHeight),
+                    0xFF5252FF, 0xFF9C9CFF, 0xFF9C9CFF, 0xFF5252FF
+                );
+            }
+            else
+            {
+                drawList.AddRectFilledMultiColor(
+                    cursorPos, cursorPos + new Vector2(barWidth * scale, BarHeight),
+                    0xFF90827C, 0xFF8E8D8F, 0xFF8E8D8F, 0xFF90827C
+                );
+            }
+
+            drawList.AddRect(cursorPos, cursorPos + barSize, 0xFF000000);
+
+        }
+
+        private void DrawSenResourceBar() {
+            var gauge = PluginInterface.ClientState.JobGauges.Get<SAMGauge>();
+
+            const int xPadding = 3;
+            const int yPadding = 3;
+            const int numChunks = 3;
+            
+            var barWidth = (BarWidth - xPadding * (numChunks - 1)) / numChunks;
+            var barSize = new Vector2(barWidth, BarHeight);
+            var xPos = CenterX - XOffset;
+            var yPos = CenterY + YOffset ;
+            var cursorPos = new Vector2(xPos - xPadding - barWidth, yPos);
+            
+            var drawList = ImGui.GetWindowDrawList();
+            
+            for (var i = 1; i < 4; i++) {
+                cursorPos = new Vector2(cursorPos.X + xPadding + barWidth, cursorPos.Y);
+                
+                if (i == 1)
+                {
+                    if (gauge.HasSetsu())
+                    {
+                        drawList.AddRectFilled(
+                        cursorPos, cursorPos + barSize,
+                        ImGui.ColorConvertFloat4ToU32(new Vector4(89 / 255f, 234 / 255f, 247 / 255f, 255f))
```

Comment:
IMO we should just directly put the color here.   If you google `rgb to hex` you'll get a little widget with an RGB box on the bottom left that you can type `89 234 247` into and it'll show you the HEX value of `#59eaf7` 

Now that you know the hexcode you can replace this line of code with 0xAABBGGRR  AA is FF for 100% opacity so your color becomes.. 0xFFF7EA59

Repeat this process for the other three colors.